lib/generators/rspec/install/templates/spec/rails_helper.rb

```diff
@@ -67,20 +67,22 @@
   # config.use_transactional_fixtures = true
 
 <% end -%>
-  # RSpec Rails can automatically mix in different behaviours to your tests
-  # based on their file location, for example enabling you to call `get` and
-  # `post` in specs under `spec/controllers`.
+  # RSpec Rails uses metadata to mix in different behaviours to your tests,
+  # for example enabling you to call `get` and `post` in request specs. e.g.:
   #
-  # You can disable this behaviour by removing the line below, and instead
-  # explicitly tag your specs with their type, e.g.:
-  #
-  #     RSpec.describe UsersController, type: :controller do
+  #     RSpec.describe UsersController, type: :request do
   #       # ...
   #     end
   #
   # The different available types are documented in the features, such as in
   # https://rspec.info/features/7-0/rspec-rails
-  config.infer_spec_type_from_file_location!
+  #
+  # You can also this infer these behaviours automatically by location, e.g.
+  # /spec/models would pull in the same behaviour as `type: :model` but this
+  # behaviour is considered legacy and will be removed in a future version.
```

Comment:
Why not keep it as a configurable option?

Reply:
This is a template, to configure it you just remove the comment part of  this line in your project.

Reply:
My concern was that this comment meant that `config.infer_spec_type_from_file_location!` will be removed in a future version

Reply:
There is no timeline for its removal, if we did we'd probably just pull it into its own gem